src/Pages/gyuyoung/Login/Login.js

```diff
@@ -1,181 +1,93 @@
 import React from 'react';
 import './Login.scss';
-import { Redirect } from 'react-router-dom';
 
 class Login extends React.Component {
 constructor(){
-    super()
-    /*
-        아니 핸들을 위에 주면 되고 아래주면 안되는이유가 뭐지??
-
-        bind차이가 뭔지 확인해볼것!
-
-        멘토님한테 물어보자
-    */
-    this.handleChange1 = this.handleChange1.bind(this);
-    this.handleChange2 = this.handleChange2.bind(this);
-    this.state ={
-        Id :'',
-        Pw :'',
-        Btn : '#b9dcfd',
-        fontId : '',
-        transformId : '',
-        fontPw : '',
-        tramsformPw : '',
-        type : 'password',
-        show : '',
-    };
+  super()
+  this.state ={
+    id :'',
+    pw :'',
+    type : 'password',
+    show : '',
+    idInInputBoxPlaceholderChanged : 'inputBoxPlaceholder',
+    pwInInputBoxPlaceholderChanged : 'inputBoxPlaceholder',
+    loginBtnClass : 'loginButton loginBtnColor',
+    idPholder : '',
+    pwPholder : '',
+  };
 }
-/*
-이벤트에 함수 적용해볼것
-
-왜 함수가 적용안된지 손으로 써볼것
-이벤트 타겟이 미적용되는거 확인
-
-[event.target.name]설정법 공부할것
-좀더 효율적인 함수 사용 조사하고 정리할것
-*/
-  handleChange1=(event)=>{
-    this.setState({Id : event.target.value});
-
-    /*
-    원하는 결과가 안됨
 
-    스테이트에서 value를 가져오면 하나씩 짤린다??
+InPutChangeHandler=(event)=>{
+  this.setState({[event.target.name] : event.target.value});
+  if(event.target.name === 'id'){
+    if(event.target.value.length>3 && this.state.pw.length>8){
+      this.setState({loginBtnClass : 'loginButton loginBtnColorChanged'})}
+    else{this.setState({loginBtnClass : 'loginButton loginBtnColor'})}
```

Comment:
```suggestion
inputChangeHandler = (event) => {
  const { name, pw, value } = event.target;
  
  this.setState({ [name]: value });
  
  if (name === 'id') {
    if(value.length > 3 && pw.length > 8) {
      this.setState({ loginBtnClass: 'loginButton loginBtnColorChanged' })
    } else {
      this.setState({ loginBtnClass: 'loginButton loginBtnColor' })
    }
```
- 띄어쓰기를 적절히 활용하여 가독성 좋은 코드를 작성해주세요.
- 대문자로 시작하는건 컴포넌트 이름입니다. 소문자로 시작해주시고 camel case로 정확히 작성해주세요.
- 구조분해 할당을 사용하면 보다 가독성 있는 코드를 작성할 수 있습니다.
- 일반적으로 클래스 명을 state에 전부 다 넣어놓는 방식을 사용하지 않습니다. state에서는 상태값을 관리하고 그 상태 값에 따라 클래스 명이 토글되도록 해주세요.
```
  <div className={`loginButton ${loginBtnClass ? "changed" : ""}`} />
```